dataframe_api_compat/polars_standard/dataframe_object.py

```diff
@@ -237,35 +237,35 @@ def __eq__(  # type: ignore[override]
         other: AnyScalar,
     ) -> DataFrame:
         return self._from_dataframe(
-            self.dataframe.with_columns(pl.col("*").__eq__(other)),
+            self.dataframe.with_columns(pl.col("*").__eq__(other)),  # type: ignore[operator]
```

Comment:
Unfortunately, I haven't found a way to fix the following errors instead of just ignoring them: `dataframe_api_compat/polars_standard/dataframe_object.py:240: error: Unsupported operand types for == ("Expr" and "Union[Union[str, int, float, bool], Scalar]")  [operator]`